lib/foreman_maintain/package_manager/dnf.rb

```diff
@@ -121,9 +121,13 @@ def info(name)
       dnf_action('module info', name, with_status: true, assumeyes: true)
     end
 
+    def package_version(package_name)
+      sys.execute!("rpm -q #{package_name} --qf '%{version}'").strip
+    end
```

Comment:
this already exists in `lib/foreman_maintain/concerns/system_helpers.rb`, no need to re-invent